responsibleai/responsibleai/modelanalysis/model_analysis.py

```diff
@@ -117,6 +133,120 @@ def __init__(self, model, train, test, target_column,
                           self._error_analysis_manager,
                           self._explainer_manager]
 
+    def _validate_model_analysis_input_parameters(
```

Comment:
What do you think about starting a new module for validation? It might be easier to test if we can break validation up into smaller functions and that's hard to do all within one class.

Reply:
Oh, just realized you had that originally.. why did you move it in the class?

Reply:
Yes, initially I had it that way. But all the validations were around the ModelAnalysis class so moved it in this class. 

 We should think about more generic validations routines that can be used in multiple places. tIf you have other similar validations that need to be added, we can discuss about moving them into a new module.